types/node/buffer.d.ts

```diff
@@ -163,6 +163,35 @@ declare module 'buffer' {
          */
         stream(): WebReadableStream;
     }
+    export interface FileOptions {
+        /**
+         * One of either `'transparent'` or `'native'`. When set to `'native'`, line endings in string source parts will be
+         * converted to the platform native line-ending as specified by `require('node:os').EOL`.
+         */
+        endings?: 'native' | 'transparent';
+        /** The File content-type. */
+        type?: string;
+        /** The last modified date of the file. `Default`: Date.now(). */
+        lastModified?: number;
+    }
+    /**
+     * A [`File`](https://developer.mozilla.org/en-US/docs/Web/API/File) provides information about files.
+     * @experimental
+     * @since v18.13.0, v19.2.0
+     */
+    export class File extends Blob {
+        constructor(sources: Array<BinaryLike | Blob>, fileName: string, options?: FileOptions);
+        /**
+         * The name of the `File`.
+         * @since v18.13.0, v19.2.0
+         */
+        name: string;
+        /**
+         * The last modified date of the `File`.
+         * @since v18.13.0, v19.2.0
```

Comment:
```suggestion
         * @since v18.13.0
```
we're not yet there on DT